spec/implementation_matrix.md

```diff
@@ -42,20 +42,20 @@
 
 |                                                      | [besu] |  erigon  | [ethereumjs] | [evmone] | [geth] | [nethermind] |  revm  | [EELS] |
 |------------------------------------------------------|--------|----------|--------------|----------|--------|--------------|--------|--------|
-| [EEST] `eip7692@v1.0.7` - `state_tests`              | ❓     |          | ✅           | ✅       |        |              | ✅     |        |
-| [EEST] `eip7692@v1.0.7` - `eof_tests`                | ❓     | ❓       | ✅           | ✅       |        |              | ✅     |        |
-| [EEST] `eip7692@v1.0.8` - `state_tests`              |        |          |              | ✅       |        | ✅           |        |        |
-| [EEST] `eip7692@v1.0.8` - `eof_tests`                |        |          |              | ✅       |        |              |        |        |
+| [EEST] `eip7692@v1.0.7` - `state_tests`              | ✅     |          | ✅           | ✅       |        |              | ✅     |        |
+| [EEST] `eip7692@v1.0.7` - `eof_tests`                | ✅     | ❓       | ✅           | ✅       |        |              | ✅     |        |
+| [EEST] `eip7692@v1.0.8` - `state_tests`              | ✅     |          |              | ✅       |        | ✅           |        |        |
+| [EEST] `eip7692@v1.0.8` - `eof_tests`                | ✅     |          |              | ✅       |        |              |        |        |
 | \[\*\] [tests] `v14.0` - `EIPTests/StateTests/stEOF` | ✅     |          |              | ✅       |        |              | ✅     |        |
 | \[\*\] [tests] `v14.0` - `EOFTests`                  | ✅     |          |              | ✅       |        |              | ✅     |        |
 | \[\*\*\] (`evmone` (old) `70ca837` - `state_tests`)  | ✅     |          | ✅           | ✅       |        |              |        |        |
 | \[\*\*\] (`evmone` (old) `70ca837` - `eof_tests`)    | ✅     |          | ✅           | ✅       |        |              |        |        |
-| [`evmone` exported] `v0.12.0` - `state_tests`        |        |          |              | ✅       |        |              | ✅     |        |
-| [`evmone` exported] `v0.12.0` - `eof_tests`          |        |          |              | ✅       |        |              | ✅     |        |
+| [`evmone` exported] `v0.12.0` - `state_tests`        | ✅     |          |              | ✅       |        |              | ✅     |        |
+| [`evmone` exported] `v0.12.0` - `eof_tests`          | ✅     |          |              | ✅       |        |              | ✅     |        |
 
 [EEST]: https://github.com/ethereum/execution-spec-tests/releases/
 [tests]: https://github.com/ethereum/tests/releases/
-[`evmone` exported]: https://github.com/ethereum/evmone/releases/
+[`evmone` exported]: https://github.com/ethereum/evmone/releases/ - except for TXCREATE and related transaction tests.
```

Comment:
thx for noting this (however these are marked with "Osaka" (meaning "some fork after Prague" ofc), so should be filtering  out